filter/fes-2.0.2/src/test/resources/org/orbisgis/orbisdata/filter/fes_2_0_2/filter_DWithIn.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<fes:Filter
+        xmlns:fes="http://www.opengis.net/fes/2.0"
+        xmlns:gml="http://www.opengis.net/gml/3.2"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.opengis.net/fes/2.0 http://schemas.opengis.net/filter/2.0/filterAll.xsd
+   http://www.opengis.net/gml/3.2 http://schemas.opengis.net/gml/3.2.1/gml.xsd">
+    <fes:DWithin>
+        <fes:ValueReference>London</fes:ValueReference>
+        <fes:ValueReference>Paris</fes:ValueReference>
+        <fes:Distance uom="m">344</fes:Distance>
+    </fes:DWithin>
+</fes:Filter>
```

Comment:
This is two files which are needed in the tests of the spatials operators but at this moment, I haven't add all the code. In the next pull request, I will add the code of spatial and Logical operator with theirs tests. At this moment, I will used this ressource file.